website/source/docs/providers/opc/r/opc_compute_storage_volume_snapshot.html.markdown

```diff
@@ -0,0 +1,58 @@
+---
+layout: "opc"
+page_title: "Oracle: opc_compute_storage_volume_snapshot"
+sidebar_current: "docs-opc-resource-storage-volume-snapshot"
+description: |-
+  Creates and manages a storage volume snapshot in an OPC identity domain.
+---
+
+# opc\_compute\_storage\_volume_snapshot
+
+The ``opc_compute_storage_volume_snapshot`` resource creates and manages a storage volume snapshot in an OPC identity domain.
+
+## Example Usage
+
+```
+resource "opc_compute_storage_volume_snapshot" "test" {
+  name        = "storageVolume1"
+  description = "Description for the Storage Volume"
+  tags        = ["bar", "foo"]
+  collocated  = true
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `volume` (Required) The name of the storage volume to create the snapshot from.
+* `description` (Optional) The description of the storage volume snapshot.
+* `name` (Optional) The name of the storage volume snapshot. Will be generated if unspecified.
+* `parent_volume_bootable` (Optional) A string value of whether or not the parent volume is 'bootable' or not. Defaults to `"false"`.
```

Comment:
It's oddly a string at the API level, https://docs.oracle.com/cloud/latest/stcomputecs/STCSA/op-storage-snapshot--post.html